docs/contributing/reporting-bugs.md

```diff
@@ -58,7 +58,7 @@ Steps and best practices for filing an issue:
    - [zulip/zulip-mobile](https://github.com/zulip/zulip-mobile/issues) for
      issues with the mobile apps.
    - [zulip/zulip-desktop](https://github.com/zulip/zulip-desktop/issues) for
-     issues that are specific to the Zulip desktop app, and _do not_ occur in
+     issues that are specific to the Zulip desktop app but _do not_ occur in
```

Comment:
I feel like this change makes this de-emphasizes this note. What about "that are specific to the Zulip desktop app, and therefore _do not_ occur in the web app."?

Reply:
I think _and_ by itself is too easy to miss, but sure, I can get behind "and therefore."